jobs/integration/conftest.py

```diff
@@ -304,9 +289,7 @@ async def fast_forward(
 
 @pytest.fixture(scope="module")
 async def tools(request):
-    tools = Tools(request)
-    await tools._load()
-    return tools
+    yield request.config.test_tools
```

Comment:
the old fixture now just has to yield the existing tools from the `request.config`.  See below in `pytest_configure`